package.json

```diff
@@ -182,6 +183,7 @@
         "@types/react-transition-group": "^4.4.0",
         "@types/sanitize-html": "2.11.0",
         "@types/sdp-transform": "^2.4.6",
+        "@types/seedrandom": "<3.0.5",
```

Comment:
this is pinned to work around https://github.com/Callidon/bloom-filters/issues/72

Reply:
This sounds like it should have been in `resolutions` rather than dev deps

Reply:
https://github.com/matrix-org/matrix-react-sdk/pull/12551 is why